src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -149,6 +149,42 @@
 	                IsChecked="{x:Bind Mode=TwoWay, Path=ViewModel.DisableDriveDetectionWarning}"
 	                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}"
 	                />
+
+            <TextBlock x:Uid="Position_GroupSettings"
+                       Style="{StaticResource SettingsGroupTitleStyle}"
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher, Converter={StaticResource ModuleEnabledToForegroundConverter}}" />
+
+            <!-- We cannot navigate to all the radio buttons using the arrow keys because of an XYNavigation issue in the RadioButtons control.
+            The screen reader does not read the heading when we tab into a radio button, even though the LabeledBy automation property is set.
+            Link to the issue in the winui repository - https://github.com/microsoft/microsoft-ui-xaml/issues/3156 -->
+            <TextBlock x:Uid="PowerLauncher_PositionModeHeader"
+                       x:Name="RadioButtons_Name_Position" 
+                       Margin="{StaticResource SmallTopMargin}"
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher, Converter={StaticResource ModuleEnabledToForegroundConverter}}" />
+            <StackPanel AutomationProperties.LabeledBy="{Binding ElementName=RadioButtons_Name_Position}">
+                <RadioButton x:Uid="Radio_Position_Cursor"
+                             IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}"
+                             IsChecked="{Binding Mode=TwoWay, Path=ViewModel.IsCursorPositionRadioButtonChecked}" />
+
+                <RadioButton x:Uid="Radio_Position_Primary"
+                             IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}"
+                             IsChecked="{Binding Mode=TwoWay, Path=ViewModel.IsPrimaryPositionRadioButtonChecked}" />
+
+                <RadioButton x:Uid="Radio_Position_Custom"
+                             IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher}"
+                             IsChecked="{Binding Mode=TwoWay, Path=ViewModel.IsCustomPositionRadioButtonChecked}" />
+
+                <muxc:NumberBox x:Uid="PowerLauncher_MonitorToDisplayOn"
```

Comment:
I think a drop-down with a predefined list (based on connected monitors) would be better here.

Reply:
Absolutely. Any idea how to query the detected monitors? I know how to do it in Win Forms with `Screen.AllMonitors` but I've yet to find the UWP equivalent.

Reply:
No. Sory. Maybe you can look at the FancyZoones project.

cc: @seraphimaz, @enricogior

Reply:
Hmm, FancyZones seems to use `Screen.AllScreens` too so I'll have to keep looking for an alternate way to retrieve it in the settings app.